update_translations.py

```diff
@@ -0,0 +1,116 @@
+import subprocess
```

Comment:
To be honest I did it this way because that's what the rest of this repository is using to do git stuff. At some point we probably should change the whole thing, but consciously didn't put in the time to investigate it now